app/deck/gateway/reset.md

```diff
@@ -23,6 +23,4 @@ related_resources:
 
 The `deck gateway reset` command deletes all entities in the target control plane.
 
-{:.danger}
```

Comment:
This was an intentional choice. It's destructive and should be highlighted with red